_posts/blog/2019-10-14-global-pilot-success.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: blog
+title: "M-Lab 2.0 Platform: Global Pilot Assessment"
+author: "Peter Boothe"
+date: 2019-10-14
+breadcrumb: blog
+categories:
+  - platform
+  - research
+  - data
+  - kernel
+  - open-source
+  - performance
+  - tcp-info
+---
+
+We deployed [the new M-Lab platform to 1/3rd of the M-Lab fleet](/data/docs/analysis/2019-10-14-pilot-assessment/), and now we
+need to assess whether or not it is a performance regression, relative to the
+old platform.  As long as we can be sure the performance of the new platform
+does not constitute a regression, then we can roll out the new platform and be
+confident that we have not made anything worse.
+
+<!--more-->
+
+Because M-Lab data is used to derive local conditions around the world, we want
+to make sure that, at every site around the world, the new platform does not
+perform worse than the old platform.  To define "worse" more explicitly, we
+first need to establish a per-site "noise floor".  Some regions of the world and
+some sites experience highly variable Internet performance, so whether a worse
+performance is statistically significant depends on the base level of variation
+that naturally occurs at that site.
+
+For every site we created a q-q plot to compare upload and dowload performance, and we also calculated the expected performance at each site.  An [ipython notebook containing the full analysis](/data/docs/analysis/2019-10-14-pilot-assessment) is available, but we reproduce one figure here:
+
+```txt
```

Comment:
I'm not sure this block is being rendered as you expect it:
http://website.mlab-sandbox.measurementlab.net/blog/global-pilot-success/